sdk/nodejs/tests/unwrap.spec.ts

```diff
@@ -51,7 +51,7 @@ function testResources(val: any, expected: any, resources: TestResource[], allRe
         assert.deepStrictEqual(asyncResources, new Set(allResources));
 
         for (const res of syncResources) {
-            if (!asyncResources.has(<TestResource>res)) {
+            if (!asyncResources.has(<Resource><any>res)) {
```

Comment:
What is the intention with this type assertion? With two type assertions here isn't this just a cast to `any`?

Reply:
This is related to https://github.com/pulumi/pulumi/pull/5311